send-templated-messages.py

```diff
@@ -26,7 +27,7 @@
 timer = Timer()
 
 try:
-    for items in csv_reader(args.csv):
+    for items in csv_reader(fh=args.csv, duplicate=args.deduplicate):
```

Comment:
very confusing naming here ? + make the deduplicate flag in the function optional defaulting to `False` as at present.

Reply:
@mzealey  Made the changes, please check the latest commit